### PR TITLE
Create a `stage` environment on AWS

### PR DESCRIPTION
**Pull request by corbt**
This PR automatically deploys a `stage` environment to AWS on each deploy to the `stage` branch. This is in preparation for moving our app to AWS from Render entirely.

### CODE REVIEW

File: app/prisma/us-west-2-bundle.pem
Comment (arcticfly) [inline app/prisma/us-west-2-bundle.pem:1]: Should this be included in the repo?

File: infra/Pulumi.prod.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+config:
+  aws-native:region: us-west-2
+  aws:region: us-west-2
+  openpipe:ANTHROPIC_API_KEY:
+    secure: AAABAG5PApLpxIrXlau1/D5VPnS7yxwv88kXog0O1T7xBxsmlcfoYWByoJ3Mw3wtM3ZXXUcrdbt/jJA5UM3Ke5Dm78lclNpoLcizDYVZ6i3+n/sRFNXXfA8musbQX6dsUAvh1Nja8mqkAp9di1IMz+eweyfHLiFPPmhrngBaD6SVc0iU9iq8dFsSPSg=
```
Comment (arcticfly) [inline infra/Pulumi.prod.yaml:5]: Are all these keys okay to include in the repo?

**Review by arcticfly — APPROVED** [review]
Thanks!

File: app/prisma/us-west-2-bundle.pem
Comment (corbt) [reply app/prisma/us-west-2-bundle.pem:1]: Yeah this is a public key from the aws site.

Only downside to including it is that it expires in August 2024 and we'll have to remember to swap it out when that gets close or else our app will stop connecting to the DB.

File: infra/Pulumi.prod.yaml (hunk 1 above)
Comment (corbt) [reply infra/Pulumi.prod.yaml:5]: Yeah they're fully encrypted, don't see a security risk here.

**Pull request merged**